test/position-managers/IncreaseLiquidity.t.sol

```diff
@@ -458,4 +464,40 @@ contract IncreaseLiquidityTest is Test, PosmTestSetup, Fuzzers {
         assertEq(currency0.balanceOf(address(this)), balanceBefore0 - amount0);
         assertEq(currency1.balanceOf(address(this)), balanceBefore1 - amount1);
     }
+
+    function test_increaseLiquidity_clearRevert() public {
```

Comment:
can you add one test where someone tries to clear a negative balance, it should revert in core